src/components/manifold-subscription-list/manifold-subscription-list.tsx

```diff
@@ -64,6 +67,43 @@ export class ManifoldSubscriptionList {
     this.getSubscriptions(this.owner);
   }
 
+  ctaHref(planID: string) {
+    if (!this.baseUrl || this.baseUrl === '#') {
+      return this.baseUrl;
+    }
+
+    // const search = new URLSearchParams();
+    // set plan ID
+    // search.set('planId', planID);
+
+    // // set configurable feature selection (or skip, if no configurable features);
+    // Object.entries(this.userSelection[planID] || {}).forEach(([key, val]) => {
+    //   search.set(key, `${val}`);
+    // });
```

Comment:
This is from the plan table and I'm having a hard time understanding what's going on here.

Reply:
It's setting some query params on the URL, you probably don't need it